index.html

```diff
@@ -161,36 +161,46 @@ <h2>Survey</h2>
             action="https://formspree.io/xyyzpdbj"
             method="POST">
 
-            <label>What is your name?<br><input type = "text" name="name_field" required></label>
+            <label for="name">What is your name?</label>
+            <input type = "text" name="name_field" id="name" required>
```

Comment:
Most of your labels were done incorrectly: you were to use the 'for' attribute on all of them except the radio button controls